README.md

```diff
@@ -124,6 +124,54 @@ The delete operation makes use of a single property ```tag``` and operates as de
 
 Note: Delete searches **all** repositories for tagged assets.
 
+# Deferred Deployment
+
+There is sometimes a desire to stage the deployment separately from performing the upload. This capability exists via
+deferred deployment for release artifacts (Snapshots are not supported). 
+
+The pom file requires configuration as described in [Example configuration](#example-configuration) with the addition of
+a ```stageLocally``` property, that can be present within the pom or on the command-line as seen in the example below.
+
+There is also the ability to use the ```altStagingDirectory``` property to specify the directory that the local staging
+will be performed within. If not specified, this will default to ```target/nexus-staging``` (and as such will be removed
+with ```mvn clean```).
+
+e.g. ```mvn clean package nxrm3:deploy -DstageLocally -DaltStagingDirectory=myStagingDir``` or
+
+```
+<plugin>
+    ...
+    <configuration>
+      ...
+      <altStagingDirectory>myStagingDir</altStagingDirectory>
+       ...
+```
+
+Within the staging directory, a suitable structure will be automatically created when locally staged, along with a
+file (```.index```) that holds metadata to allow subsequent uploading. The staging directory may be moved elsewhere and
+subsequent uploads performed from the new location, so long as the new directory location is specified through the 
+```altStagingDirectory``` property on the upload command-line.
+
+For example: ```mvn nxrm3:upload ... -DaltStagingDirectory=<directory>```
+
+### Deferred Deployment Example
+
+Stage the deployment, but defer the upload:
+
+```mvn clean package nxrm3:deploy -DstageLocally```
+
+The staging directory may now be moved or left in place - it is important that nothing within this directory structure
+is modified externally, to guarantee the integrity of the deployment, and that appropriate ```altStagingDirectory```
+properties are provided where necessary.
+
+At a later time:
```

Comment:
@mobiuscog unsure what this means...like you should use this command at a later time or something else?

Reply:
Yes - the deferred deployment is composed of two actions, one to stage locally, and then the other to upload what was previously staged.